ReSwift-Thunk/createThunksMiddleware.swift

```diff
@@ -23,3 +23,10 @@ public func createThunksMiddleware<State: StateType>() -> Middleware<State> {
         }
     }
 }
+
+// swiftlint:disable identifier_name
+@available(*, deprecated, renamed: "createThunksMiddleware")
+func ThunkMiddleware<State: StateType>() -> Middleware<State> {
```

Comment:
Thanks for merging everything, @mjarvis! I have to say, I like `ThunkMiddleware<T>()` more, now that I see it in context ;)

Reply:
I sort of like it too, but it breaks swift guidelines for naming -- It makes it look like its an object (struct/class/enum), instead of a function, because of the uppercase `T`.